common.blocks/error-board/error-board.bemtree.js

```diff
@@ -0,0 +1,33 @@
+block('error-board').content()(function() {
+    return [
+        {
+            elem: 'logo'
+        },
+        {
+            elem: 'description',
+            content:[
+                {
+                    elem: 'title',
+                    tag: 'h1',
+                    content : 'Страница не найдена'
```

Comment:
Please use `i18n` the same way as in https://github.com/tadatuta/bem-forum/pull/16/files#diff-debffe068b0807942ac1a47903381ca2R1